src/overlays/actors/ovl_Dm_Char04/z_dm_char04.h

```diff
@@ -8,10 +8,16 @@ struct DmChar04;
 typedef void (*DmChar04ActionFunc)(struct DmChar04*, PlayState*);
 
 typedef struct DmChar04 {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0xF8];
-    /* 0x23C */ DmChar04ActionFunc actionFunc;
-    /* 0x240 */ char unk_240[0x24];
+    /* 0x0000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[15];
+    /* 0x1E2 */ Vec3s morphTable[15];
+    /* 0x023C */ DmChar04ActionFunc actionFunc;
+    /* 0x240 */ Color_RGBAf primColors;
+    /* 0x250 */ Color_RGBAf envColors;
+    /* 0x260 */ u8 csAction;
+    /* 0x261 */ u8 animationIndex;
+    /* 0x262 */ u16 unk_262;
```

Comment:
This is definitely a timer that seems to offset limbs and opacity over time. We also should have a macro for it's access.